tests/plugins/autolink/autolink.js

```diff
@@ -41,6 +40,12 @@
 
 	bender.test( {
 
+		setUp: function() {
+			if ( !CKEDITOR.plugins.registered.autolink.isSupportedEnvironment() ) {
```

Comment:
There is a nice bender alias for this: `bender.tools.ignoreUnsupportedEnvironment( 'autolink' );` (used e.g. [here](https://github.com/ckeditor/ckeditor4/blob/major/tests/plugins/autocomplete/autocomplete.js#L41)). This should be replaced also in all manual tests.

Reply:
Updated: https://github.com/ckeditor/ckeditor4/pull/2518/commits/1102077485b8c4f75ceeb337f143fce444c88c87 and https://github.com/ckeditor/ckeditor4/pull/2518/commits/8bea34024c8ca1039d85b0743ac6dfc943c1dacc and https://github.com/ckeditor/ckeditor4/pull/2518/commits/14ae36c2daf708739665e3b322c53b797dd61745